src/main/scala/com/codacy/rules/ReportRules.scala

```diff
@@ -72,6 +72,14 @@ class ReportRules(coverageServices: => CoverageServices) extends LogSupport {
           sendFilesReportForCommit(files, config, partial = config.partial, commitUUID)
         }
       }
+      if (config.partial) {
+        logger.info(
+          """To complete the reporting process, call the coverage-reporter with a final flag.
+          | Check https://github.com/codacy/codacy-coverage-reporter#multiple-coverage-reports-for-the-same-language
+          | for more information.""".stripMargin
```

Comment:
Suggestion a minor tweak to the copy of the message:

```suggestion
          """To complete the reporting process, call coverage-reporter with the final flag.
          | Check https://github.com/codacy/codacy-coverage-reporter#multiple-coverage-reports-for-the-same-language
          | for more information.""".stripMargin
```